projects/rocprofiler-sdk/source/lib/python/rocpd/summary.py

```diff
@@ -53,13 +53,12 @@ def make_temp_view_query(view_name, query) -> str:
     return "CREATE TEMPORARY VIEW IF NOT EXISTS `{}` AS {}".format(view_name, query)
```

Comment:
```suggestion
def make_temp_view_query(view_name, query) -> str:
    return "CREATE TEMPORARY VIEW IF NOT EXISTS `{}` AS {}".format(view_name, query)
```

Is there a way we can remove all uses of this "make_temp_view_query"?  Seems like it's a lot more work to do that, but I think that's the challenge Jonathan posed....

Reply:
Removed the function "make_temp_view_query"